### PR TITLE
Adjusted delay for update history wait

### PR DESCRIPTION
**Pull request by KolombetM**
One of the async methods did not have enough time to save results into ruleHistory

### CODE REVIEW

File: src/foam/nanos/ruler/test/RulerDAOTest.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ public void testRuleHistory(X x) {
 
     // wait for async
     try {
-      Thread.sleep(asyncWait + 100);
+      Thread.sleep(asyncWait + 2000);
```
Comment (jlhughes) [inline src/foam/nanos/ruler/test/RulerDAOTest.java]: Thinking perhaps a better option is to loop for max amount of time.   so loop, delay 100, for a max of 2000 for example.

Comment (KolombetM) [reply src/foam/nanos/ruler/test/RulerDAOTest.java]: You don't seem to get out of the loop unless you finish all the iterations.
Screen Shot 2020-02-05 at 2 27 45 PM

Comment (jlhughes) [reply src/foam/nanos/ruler/test/RulerDAOTest.java]: move the find into the loop, exit early if not null.

File: src/foam/nanos/ruler/test/RulerDAOTest.java [hunk 2]
```diff
@@ -88,9 +88,15 @@ public void testRuleHistory(X x) {
 
     // wait for async
     try {
-      Thread.sleep(asyncWait + 100);
+      for (int i = 0; i < 2000; i++) {
+        Thread.sleep(asyncWait + 100);
+        ruleHistory = (RuleHistory) ruleHistoryDAO.find(ruleHistory.getId());
+        if ( ruleHistory.getResult().equals("Pending") )
```
Comment (KolombetM) [inline src/foam/nanos/ruler/test/RulerDAOTest.java]: Since ruleHistory in the pending state, until it is done, we wait for the state to be changed.

**Review by jlhughes — APPROVED** [review]

**Pull request merged**